docs/Documentation/UserGuide.md

```diff
@@ -0,0 +1,24 @@
+# RBM Model User Guide
+
+Running the RBM model may seem a formidable task, but approaching the task in steps will ease the process. In general, running VIC successfully involves preparation of different input files, downloading the source code, configuring output files, and entering a few commands. Once run, VIC output must be entered into a routing model in order to produce a hydrograph. A [sample data set for the Stehekin Basin](ftp://ftp.hydro.washington.edu/pub/HYDRO/models/VIC/Datasets/vic.sample.stehekin.tgz) is available as an example in the Sample Data Sets section of the [download page](../SourceCode/Code.md).
```

Comment:
I would delineate this page a bit better - since this first paragraph is about running VIC (not RBM) - so if this was under the "Running VIC" subheading, it would be more clear.